sdcm/stress_thread.py

```diff
@@ -136,9 +136,12 @@ def _run_cs_stress(self, loader, loader_idx, cpu_idx, keyspace_idx):  # pylint:
         if self.stress_num > 1:
             cpu_options = f'--cpuset-cpus="{cpu_idx}"'
 
-        docker = RemoteDocker(loader, "scylladb/scylla:latest",
-                              extra_docker_opts=f'{cpu_options} -v /etc/scylla/ssl_conf:/etc/scylla/ssl_conf '
-                                                f'--label shell_marker={self.shell_marker}')
+        if loader.is_kubernetes():
+            docker = loader.remoter
+        else:
+            docker = RemoteDocker(loader, "scylladb/scylla:latest",
+                                  extra_docker_opts=f'{cpu_options} -v /etc/scylla/ssl_conf:/etc/scylla/ssl_conf '
+                                                    f'--label shell_marker={self.shell_marker}')
```

Comment:
host networking?

Reply:
See few lines above, for c-s on k8s we don't use the  RemoteDocker we just use the default loader pod (which is already matching the scylla version)

I could add host networking, but it won't change much, since we aren't going to use it for k8s